Documentation/ApiOverview/SiteHandling/SiteSettingDefinitions.rst

```diff
@@ -8,88 +8,187 @@ Site settings definitions
+
+..  contents:: Table of contents
+
+..  _site-settings-definition-example:
+
+Site setting definition example
+===============================
+
+..  literalinclude:: _Settings/_blog_settings.definitions.yaml
+    :caption: EXT:blog_example/Configuration/Sets/BlogExample/settings.definitions.yaml (Excerpt)
+
+See the complete example at
+`settings.definitions.yaml (GitHub) <https://github.com/TYPO3-Documentation/blog_example/blob/main/Configuration/Sets/BlogExample/settings.definitions.yaml>`__.
+
+..  figure:: /Images/ManualScreenshots/SiteHandling/SiteSettingsDefinition.png
+    :alt: Screenshot demonstration the position of the categories, labels etc
+
+    The parts marked by a number can be configured, see list bellow
+
+
+..  _site-settings-definition-properties:
+
+Site setting definition properties
+==================================
+
+..  confval-menu::
+    :display: table
+    :name: site-setting-definition
+    :type:
+    :required:
+
+    ..  confval:: categories
+        :type: array
+        :name: site-settings-definition-categories
+
+        ..  confval:: label
+            :type: string
+            :name: site-settings-definition-categories-label
+
+        ..  confval:: parent
+            :type: :confval:`site-settings-definition-categories` key
+            :name: site-settings-definition-categories-parent
+
+    ..  confval:: settings
+        :type: array
+        :name: site-settings-definition-settings
+
+        ..  confval:: label
+            :type: string
+            :name: site-settings-definition-settings-label
+
+        ..  confval:: description
+            :type: string
+            :name: site-settings-definition-settings-description
+
+        ..  confval:: category
+            :type: :confval:`site-settings-definition-categories` key
+            :name: site-settings-definition-settings-category
+
+        ..  confval:: type
+            :type: a :ref:`definition type <definition-types>`
+            :name: site-settings-definition-settings-type
+            :required:
+
+        ..  confval:: default
+            :type: mixed
+            :name: site-settings-definition-settings-default
+            :required:
+
+            The default value must have the same type like defined in
+            :confval:`site-settings-definition-settings-type`.
+
+        ..  confval:: readonly
+            :type: bool
+            :name: site-settings-definition-settings-readonly
+
+            If a site setting is marked as readonly, it can be overridden only
+            by editing  the :file:`config/sites/my-site/settings.yaml` directly,
+            but not from within the editor.
 
 ..  _definition-types:
 
 Definition types
 ================
 
-..  confval:: int
-    :name: site-setting-type-int
-    :type: string
-    :Path: settings.[my_val].type = int
+..  confval-menu::
+    :display: table
+    :name: site-setting-type
+    :type:
+    :required:
```

Comment:
You add column "required", but not marked any following config as "required". Can be removed then, right?

Reply:
This is displayed correctly when I render it: 
![image](https://github.com/user-attachments/assets/a42b40a2-1de3-4af7-b21f-035ff4108f4f)